### ISSUE TITLE
Doesn`t work on WSL

### ISSUE THREAD

**Issue by codeleta** [NikitaBeloglazov/clipman#10]
I have Windows 11 + WSL2 (Ubuntu)
When i run `clipman.init()` i have an error

```
raise exceptions.NoEnginesFoundError(error_message)
clipman.exceptions.NoEnginesFoundError: The graphical backend (X11, Wayland) or running KDE was not found on your Linux OS. Check XDG_SESSION_TYPE variable in your ENV. Also, please note that TTY is unsupported.

XDG_SESSION_TYPE content: < NOT SET >
```

**Comment by NikitaBeloglazov** [comment]
Hello!
Please show ```clipman.init(debug=True)```

**Comment by codeleta** [comment]
```
>>> clipman.init(debug=True)
init call start
display_server: < NOT SET >
current_desktop: < NOT SET >
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/codeleta/.cache/pypoetry/virtualenvs/deploy-notify-vYaV6AjP-py3.9/lib/python3.9/site-packages/clipman/__init__.py", line 300, in init
    dataclass.engine = detect_clipboard_engine()
  File "/home/codeleta/.cache/pypoetry/virtualenvs/deploy-notify-vYaV6AjP-py3.9/lib/python3.9/site-packages/clipman/__init__.py", line 189, in detect_clipboard_engine
    raise exceptions.NoEnginesFoundError(error_message)
clipman.exceptions.NoEnginesFoundError: The graphical backend (X11, Wayland) or running KDE was not found on your Linux OS. Check XDG_SESSION_TYPE variable in your ENV. Also, please note that TTY is unsupported.

XDG_SESSION_TYPE content: < NOT SET >

```

**Comment by NikitaBeloglazov** [comment]
Hmm. Seems in pyperclip is it broken:)

Well, I found a solution on how to do this in clipman. 
I just need to write the code when will i get some free time

**Comment by NikitaBeloglazov** [comment]
## ✅ Fixed in [v3.2.1](https://github.com/NikitaBeloglazov/clipman/releases/tag/v3.2.1)

https://github.com/NikitaBeloglazov/clipman/releases/tag/v3.2.1

_**I honestly don't recommend using WSL. Please consider use alternative solutions for your needs**_
Thank you for issue!
